cmd/cmd.go

```diff
@@ -33,6 +33,17 @@ import (
 	"github.com/jmorganca/ollama/version"
 )
 
+type Painter struct{}
+
+func (p Painter) Paint(line []rune, l int) []rune {
+	termType := os.Getenv("TERM")
+	if termType == "xterm-256color" && len(line) == 0 {
+		prompt := "Send a message (/? for help)"
+		return []rune(fmt.Sprintf("\033[38;5;245m%s\033[%dD\033[0m", prompt, len(prompt)))
```

Comment:
what do these magical formatting numbers do?

Reply:
Just to break it down:

`\033[38;5;245m` - turns the text colour to grey
`%s` - print out the "Send a message"
`\033[%dD` - move the cursor back %d chars
`\033[0m` - reset the color to default

the grey is neutral, so should work in either dark or light themed terminals.